### ISSUE TITLE
Google Colab notebook for scene segmentation on Scannet - problem with passing dataset to model

### ISSUE THREAD

**Issue by nikolamilovic1** [torch-points3d/torch-points3d#788]
Hi,

Inspired by [given notebooks](https://github.com/torch-points3d/torch-points3d/tree/master/notebooks), I have tried to implement scene segmentation on **Scannet** dataset using **KPConv**. I tried to follow the steps from **[PartSegmentationKPConv](https://github.com/torch-points3d/torch-points3d/blob/master/notebooks/PartSegmentationKPConv.ipynb)** but when I get to the point where my model needs to be instantiated, the dataset's categories are null 
(output for the command `vars(dataset)` is:
`{'dataset_opt': {'class': 'scannet.ScannetDataset', 'task': 'segmentation', 'version': 'v2', 'dataroot': 'TempDIRECTORY/data', 'first_subsampling': 0.02, 'pre_transforms': [{'transform': 'NormalizeScale'}, {'transform': 'GridSampling3D', 'params': {'size': '${first_subsampling}'}}], 'train_transforms': [{'transform': 'RandomNoise', 'params': {'sigma': 0.01, 'clip': 0.05}}, {'transform': 'RandomScaleAnisotropic', 'params': {'scales': [0.9, 1.1]}}, {'transform': 'AddOnes'}, {'transform': 'AddFeatsByKeys', 'params': {'list_add_to_x': [True], 'feat_names': ['ones'], 'delete_feats': [True]}}], 'test_transforms': [{'transform': 'AddOnes'}, {'transform': 'AddFeatsByKeys', 'params': {'list_add_to_x': [True], 'feat_names': ['ones'], 'delete_feats': [True]}}]},
 '_data_path': 'TempDIRECTORY/data/scannet',
 '_batch_size': None,
 'strategies': {},
 '_contains_dataset_name': False,
 'train_sampler': None,
 'test_sampler': None,
 'val_sampler': None,
 '_train_dataset': Scannet(1201),
 '_test_dataset': [Scannet(100)],
 '_val_dataset': Scannet(312),
 'train_pre_batch_collate_transform': None,
 'val_pre_batch_collate_transform': None,
 'test_pre_batch_collate_transform': None,
 'pre_transform': Compose([
     NormalizeScale(),
     GridSampling3D(grid_size=0.02, quantize_coords=False, mode=mean),
 ]),
 'test_transform': Compose([
     AddOnes(),
     AddFeatsByKeys(ones=True),
 ]),
 'train_transform': Compose([
     RandomNoise(sigma=0.01, clip=0.05),
     RandomScaleAnisotropic([0.9, 1.1]),
     AddOnes(),
     AddFeatsByKeys(ones=True),
 ]),
 'val_transform': None,
 'inference_transform': Compose([
     NormalizeScale(),
     GridSampling3D(grid_size=0.02, quantize_coords=False, mode=mean),
     AddOnes(),
     AddFeatsByKeys(ones=True),
 ]),
 'pre_filter': None,
 'used_properties': {'class_to_segments': None}}`)

![image](https://user-images.githubusercontent.com/86633801/193127171-5f7d7d7e-db65-4a38-a6e8-2d0c15e140b1.png)

I tried to find out how this is done in official repository but it seemed pretty hard to find the moment where model is instantiated with dataset categories as argument.
If anyone has the idea how I could accomplish **scene segmentation** using **Scannet** in **Google Colab** (**I don't want to do this by downloading repository and running `poetry run python train.py task=segmentation models=...`. I need to create Google Colab notebook for this**) - it would be awesome. I guess this notebook could be useful for everyone. Also, if KPConv is not the best option for scene segmentation on Scannet, then I would go in that direction. KPConv is not necessary.
Thanks in advance!

My Google Colab notebook is in attachment.
*Note: I installed torch points 3D inside google drive folder so I don't need to wait for hours for every time I import necessary packages in Google Colab.

[Scannet_scene_segmentation_example.zip](https://github.com/torch-points3d/torch-points3d/files/9679350/Scannet_scene_segmentation_example.zip)
